File: src/main/java/org/c3lang/intellij/psi/C3ConstdefConstantMixin.java
package org.c3lang.intellij.psi;

import com.intellij.psi.StubBasedPsiElement;
import org.c3lang.intellij.stubs.C3ConstdefConstantStub;
import org.jetbrains.annotations.NotNull;

public interface C3ConstdefConstantMixin extends StubBasedPsiElement<C3ConstdefConstantStub>, C3PsiNamedElement, C3NameIdentProvider, C3TypeFullyQualifiedNamePsiElement
{
	@NotNull String getConstIdent();
}

File: src/main/java/org/c3lang/intellij/psi/C3ConstdefConstantMixin.kt
package org.c3lang.intellij.psi

import com.intellij.psi.StubBasedPsiElement
import org.c3lang.intellij.stubs.C3ConstdefConstantStub

interface C3ConstdefConstantMixin :
    StubBasedPsiElement<C3ConstdefConstantStub>,
    C3PsiNamedElement,
    C3NameIdentProvider,
    C3TypeFullyQualifiedNamePsiElement {
    val constIdent: String
}
